docs/config/README.md

```diff
@@ -21,7 +21,7 @@ TLS certificates by default are stored under ``${HOME}/.minio/certs`` directory.
 Following is the directory structure for MinIO server with TLS certificates.
 
 ```sh
-$ mc tree --files ~/.minio
+$ tree ~/.minio
```

Comment:
Why is this not correct? 

```
~ mc tree --files ~/.minio
/home/harsha/.minio
├─ CAs
├─ certs
│  ├─ CAs
│  ├─ private.key.old
│  └─ public.crt
├─ config.json.deprecated
├─ config.json.deprecated.old
└─ config.json.old
```

Reply:
I got error when executing the command
```

Failed to run:
Unknown option --files

Usage:
  mc [OPTION…] [+number] [this_dir] [other_panel_dir]
 +number - Set initial line number for the internal editor


GNU Midnight Commander 4.8.19


Help Options:
  -h, --help                Show help options
  --help-all                Show all help options
  --help-terminal           Terminal options
  --help-color              Color options

Application Options:
  -V, --version             Displays the current version
  -f, --datadir             Print data directory
  -F, --datadir-info        Print extended info about used data directories
  --configure-options       Print configure options
  -P, --printwd=<file>      Print last working directory to specified file
  -U, --subshell            Enables subshell support (default)
  -u, --nosubshell          Disables subshell support
  -l, --ftplog=<file>       Log ftp dialog to specified file
  -v, --view=<file>         Launches the file viewer on a file
  -e, --edit=<file> ...     Edit files


Please send any bug reports (including the output of 'mc -V')
as tickets at www.midnight-commander.org
```

Reply:
Oh wait, I think we are talking about different "mc"

Reply:
yes @oneoneonepig

Reply:
Got it! Sorry for that, I just got linked from somewhere else. I'll give the new "mc" a try 😀